xLights/TabSetup.cpp

```diff
@@ -1575,6 +1575,8 @@ void xLightsFrame::InitialiseControllersTab(bool rebuildPropGrid) {
         List_Controllers->EnsureVisible(itemSelected);
     }
 
+    Panel2->SetMinSize(wxSize(400, -1));
```

Comment:
this is your secret Controller Panel change to prevent squished buttons....

Reply:
Yes ... not so secret. How is it working for you? Isnt it always collapsed into nothing?
![image](https://github.com/xLightsSequencer/xLights/assets/4643499/efddf56b-f7c3-4491-9507-16ccbfeb4a9b)

Reply:
Enhancement request :-) https://github.com/xLightsSequencer/xLights/issues/4443